artiq/frontend/artiq_coremgmt.py

```diff
@@ -132,9 +140,25 @@ def main():
             for key, filename in args.file:
                 with open(filename, "rb") as fi:
                     mgmt.config_write(key, fi.read())
+        if args.action == "write_channel_name":
+            ddb = DeviceDB(args.device_db).get_device_db()
+            for device, value in ddb.items():
+                try:
+                    channel = value["arguments"]["channel"]
+                    mgmt.config_write("channel "+str(channel), device.encode("utf-8"))
```

Comment:
Factor this into a function that takes a DDB and returns a channel number->name dictionary.